### PR TITLE
[Does]Fix the get_started.md and yolov5_tutorial.md

### PR DESCRIPTION
**Pull request by Zheng-LinXiao**
Fix the get_started.md and yolov5_tutorial.md

### CODE REVIEW

**Comment by hhaAndroid** [comment]
@Zheng-LinXiao Thanks a lot for your contribution, please fix the ci. 

Please run `pre-commit run --all-files` on your local machine

**Comment by vansin** [comment]
```shell
cd mmyolo
pip install pre-commit
pre-commit install
pre-commit run --all-files
```

File: docs/zh_cn/user_guides/yolov5_tutorial.md [hunk 1]
```diff
@@ -14,10 +14,14 @@ mim install mmengine
 mim install "mmcv>=2.0.0rc1"
 mim install "mmdet>=3.0.0rc0"
```
Comment (PeterH0323) [inline docs/zh_cn/user_guides/yolov5_tutorial.md:15]:
```suggestion
mim install "mmcv>=2.0.0rc1,<2.1.0" 
mim install "mmdet>=3.0.0rc1,<3.1.0"
```

File: docs/zh_cn/user_guides/yolov5_tutorial.md [hunk 2]
```diff
@@ -14,10 +14,14 @@ mim install mmengine
...
 # for albumentations
```
Comment (PeterH0323) [inline docs/zh_cn/user_guides/yolov5_tutorial.md:16]:
```suggestion
```

**Review by PeterH0323 — CHANGES_REQUESTED** [review]

**Review by hhaAndroid — APPROVED** [review]

**Pull request merged**